intellij-setup.md

```diff
@@ -28,23 +28,27 @@ Android Studio (and various other JetBrains editors) is currently not supported.
 
 ### Installing the plugins
 
-We provide two plugins:
+To use IntelliJ with Flutter, you must install two plugins:
 
   * The `Dart` plugin offers code analysis (code validation as you type, code completions, etc.).
   * The `Flutter` plugin powers Flutter developer workflows (running, debugging, hot reload, etc.).
 
 To install the plugins:
 
-1. Open plugin preferences (**Preferences>Plugins** on macOS, **File>Settings>Plugins** on Linux)
-1. Select **"Browse repositories…"**
-1. Search for `'dart'`, select the Dart plug-in and click `install` (do not restart yet).
+1. Start IntelliJ.
+1. Open plugin preferences (**Preferences>Plugins** on macOS, **File>Settings>Plugins** on Linux).
+1. Select **Install JetBrains plugin...**, select the Dart plug-in and click `install` (do not restart yet).
   *  Note: this is pre-installed in WebStorm.
-1. Search for `'flutter'`,  select the Flutter plug-in and click `install`. 
+1. Select **Browse repositories…**,  select the Flutter plug-in and click `install`. 
 1. Click `Restart IntelliJ IDEA`.
 
-### Configuring the plugins
+After restarting, the Dart and Flutter plugins should both display in the left
+navigation panel when you create a new project.
 
-1. Open preferences (**Preferences** on macOS, **File>Settings** on Linux)
-1. Select **Languages & Frameworks>Flutter**
-1. Enter, or browse to, your Flutter SDK directory path in **Flutter SDK path**
+### Configuring the Flutter plugin
+
+1. Open preferences (**Preferences** on macOS, **File>Settings** on Linux).
+1. Select **Languages & Frameworks>Flutter**.
+1. Enter or browse to your Flutter SDK directory in **Flutter SDK path**
```

Comment:
People need to know how to find the Flutter SDK path (until we can find it automatically). We can either do this by telling them when they git clone it that this is the SDK, or here that they should look for the place where they git cloned it into. I'll leave it up to your discretion as to which.